coderd/workspacebuilds.go

```diff
@@ -25,6 +25,11 @@ import (
 	"github.com/coder/coder/codersdk"
 )
 
+var (
+	errTemplateNotExists = xerrors.New("No template exists for this workspace")
+	errUserNotExists     = xerrors.New("User does not exist")
```

Comment:
Nit: Go errors [should be lower case](https://github.com/golang/go/wiki/CodeReviewComments#error-strings). Also, unless needed to check against somewhere for the specific error, consider returning inline as they will then include a meaningful stack trace.

Reply:
Fixed!